plugins/modules/inventory_intent.py

```diff
@@ -2465,18 +2478,24 @@ def get_provision_wired_device(self, device_ip):
             logs the response.
         """
 
-        response = self.dnac._exec(
-            family="sda",
-            function='get_provisioned_wired_device',
-            op_modifies=True,
-            params={"device_management_ip_address": device_ip}
-        )
+        try:
+            flag = 3
+            response = self.dnac._exec(
+                family="sda",
+                function='get_provisioned_wired_device',
+                op_modifies=True,
+                params={"device_management_ip_address": device_ip}
+            )
 
-        if response.get("status") == "failed":
-            self.log("Cannot do provisioning for wired device {0} because of {1}.".format(device_ip, response.get('description')), "ERROR")
-            return False
+            if response.get("status") == "success" and "retrieved successfully" in response.get("description"):
```

Comment:
Can we match  the string instead of comparing?

Reply:
not needed.